src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -270,7 +270,8 @@ class ObjectField extends React.Component {
             [objectFields[0], 'ui:options', 'expandUnder'],
             uiSchema,
           )
-            ? renderProp(objectFields[0], index)
+            ? // eslint-disable-next-line sonarjs/no-extra-arguments
```

Comment:
Since it's not clear why the function is taking 2 params instead of 1, this rule was disabled to be in compliance with the activation of the rule. I disabled it to prevent that the changes might break the existing form system.